OpenEdXMobile/res/layout/panel_google_button.xml

```diff
@@ -22,12 +22,11 @@
             android:orientation="horizontal">
 
             <ImageView
-                android:layout_width="15dp"
-                android:layout_height="15dp"
-                android:layout_marginLeft="12dp"
-                android:layout_marginRight="12dp"
+                android:layout_width="24dp"
+                android:layout_height="24dp"
+                android:layout_margin="8dp"
```

Comment:
All these dimensions and the same dimensions in `panel_facebook_button.xml` should be standardized.